From d273f13c89b708a3cfcfddb11553dab6e2d17d20 Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:26:35 +0200
Subject: [PATCH] formatting: use tabs instead of spaces

Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
Signed-off-by: Rostislav Koval <kovalr2000@gmail.com>
---
 calculator.cpp | 6 +++++-
 calculator.h   | 6 +++---
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index 498f7b9..ad3394d 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,10 +2,14 @@
 
 int Calculator::Add (double a, double b)
 {
+<<<<<<< HEAD
     return a + b + 0.5;
+=======
+	return a + b;
+>>>>>>> cbcb06d... formatting: use tabs instead of spaces
 }
 
 int Calculator::Sub (double a, double b)
 {
-    return Add (a, -b);
+	return Add (a, -b);
 }
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.20.1

